modules/uploads/scripts/delete.ts

```diff
@@ -0,0 +1,78 @@
+import { RuntimeError, ScriptContext } from "../_gen/scripts/delete.ts";
+import { getKey } from "../utils/types.ts";
+import { deleteKeys } from "../utils/bucket.ts";
+
+export interface Request {
+	uploadId: string;
+}
+
+export interface Response {
+	bytesDeleted: string;
+}
+
+export async function run(
```

Comment:
On all of these— this would be called with `ctx.modules.uploads.deleteUpload({ ... })`, which seems slightly redundant to me at least.

IMO, `ctx.modules.uploads.delete({ ... })` seems to be both no more ambiguous and shorter.

Reply:
Additionally:
- list
  - `ctx.modules.uploads.listUploadsForUser({ ... })`
  - `ctx.modules.uploads.listForUser({ ... })`
- prepare
  - `ctx.modules.uploads.prepareUpload({ ... })`
  - `ctx.modules.uploads.prepare({ ... })`
- get
  - `ctx.modules.uploads.getUpload({ ... })`
  - `ctx.modules.uploads.get({ ... })`
- complete
  - `ctx.modules.uploads.completeUpload({ ... })`
  - `ctx.modules.uploads.complete({ ... })`

I don't think anyone will confuse `...modules.uploads.prepare()` for preparing anything but an upload.

Reply:
Resolved as per https://rivet-gg.slack.com/archives/C04E356KEP8/p1713401041625769?thread_ts=1713396640.587149&cid=C04E356KEP8